shared/profile/render.native.js

```diff
@@ -27,13 +31,59 @@ class Render extends Component<void, Props, State> {
     super(props)
     this.state = {
       currentFriendshipsTab: 'Followers',
+      activeMenuProof: null,
     }
   }
 
   _renderComingSoon () {
     return <ComingSoon />
   }
 
+  _handleToggleMenu (idx: number) {
+    const selectedProof = this.props.proofs[idx]
+    this.setState({
+      activeMenuProof: this.state.activeMenuProof && this.state.activeMenuProof.id === selectedProof.id ? undefined : selectedProof,
+    })
+  }
+
+  _proofMenuContent (proof: Proof) {
+    if (proof.meta === metaUnreachable) {
+      return {
+        header: {title: 'Your proof could not be found, and Keybase has stopped checking. How would you like to proceed?', danger: true},
+        items: [
+          ...(proof.humanUrl ? [{title: 'View proof', onClick: () => this.props.onViewProof(proof)}] : []),
+          {title: 'I fixed it - recheck', onClick: () => this.props.onRecheckProof(proof)},
+          {title: 'Revoke proof', danger: true, onClick: () => this.props.onRevokeProof(proof)},
+        ],
+      }
+    }
+    if (proof.meta === metaPending) {
+      let pendingMessage
+      if (proof.type === 'hackernews') {
```

Comment:
Though a lot of it is shared between Desktop and Mobile, there are some discrepancies in wording and options, and there are large differences in headings.